packages/cln_plugin/lib/src/json_rpc/response.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'error.dart';
+
+part 'response.g.dart';
+
+@JsonSerializable(explicitToJson: true)
+class Response {
+  /// A String specifying the version of the JSON-RPC protocol.
+  /// MUST be exactly "2.0".
+  String jsonrpc;
+
+  /// This structure contains additional information about the
+  /// result returned from the function.
+  dynamic result;
```

Comment:
```suggestion
  Map<String, dynamic> result;
```

Only the dynamic is wrong, because is like "I don'n know what I'm doing but it compiles :)"

Reply:
Ohh okayy!! Making the field nullable will look out for all this, i thought of doing so but I was worried it may expose code generation prone to errors with enabling null fields, or maybe API  issuing nulls, etc.

Reply:
well we try to hide the complexity in out API :)